cmd/boost/wallet_cmd.go

```diff
@@ -103,15 +110,30 @@ var walletList = &cli.Command{
 		// Assume an error means no default key is set
 		def, _ := n.Wallet.GetDefault()
 
-		tw := tablewriter.New(
-			tablewriter.Col("Address"),
-			tablewriter.Col("ID"),
-			tablewriter.Col("Balance"),
-			tablewriter.Col("Market(Avail)"),
-			tablewriter.Col("Market(Locked)"),
-			tablewriter.Col("Nonce"),
-			tablewriter.Col("Default"),
-			tablewriter.NewLineCol("Error"))
+		// Map Keys. Corresponds to the standard tablewriter output
+		addressKey := "Address"
+		idKey := "ID"
+		balanceKey := "Balance"
+		marketAvailKey := "Market(Avail)"
+		marketLockedKey := "Market(Locked)"
+		nounceKey := "Nounce"
+		defaultKey := "Default"
+		errorKey := "Error"
+
+		// One-to-one mapping between tablewriter keys and JSON keys
+		tableKeysToJsonKeys := map[string]string{
+			addressKey:      strings.ToLower(addressKey),
+			idKey:           strings.ToLower(idKey),
+			balanceKey:      strings.ToLower(balanceKey),
+			marketAvailKey:  "marketAvail",
+			marketLockedKey: "marketLocked",
```

Comment:
Can we move these into their own object:
{
  "market": { "available": "1234", "locked": "1234" }
}

Reply:
It makes total sense to me to do what you suggested. I implemented it.
I am not very happy about the implementation. Because the structure is different between json and tablewriter, the code is clumsier. I would have to refactor heavily to make it prettier, but I don't think it's worth it.

Reply:
Yeah it's not a big deal, I think it's fine how you did it 👍